data/common.yaml

```diff
@@ -0,0 +1,14 @@
+---
+profile_mysql_server::create_mysql_home: true
+profile_mysql_server::dbs: {}
+profile_mysql_server::manage_yumrepo: false
+profile_mysql_server::mysql_gid: null
+profile_mysql_server::mysql_groupname: "mysql"
+profile_mysql_server::mysql_home: "/var/lib/mysql"
+profile_mysql_server::mysql_logdir: "/var/log/mariadb"
+profile_mysql_server::mysql_parentdir: null
```

Comment:
I think this parent parameter can be removed?

Reply:
Done